CONSTITUTION.md

```diff
@@ -392,7 +392,7 @@ complexity demands more intracellular security.)
 Imagine how easy it would be to create a fund and simply buy VISA… well not
 even buy, but simply bond the capital of the market cap of VISA, of $391B.
 That’s a lot of money, but if bonding that capital means one can take control
-over the financial system, people would lend their money in a heartbeat. But
+of the financial system, people would lend their money in a heartbeat. But
```

Comment:
maybe, but `over` is a bit more impactful here, simbolizing a top-down strings-pulling. 
Maybe I would leave it as is